Extensions/Physics3DBehavior/Physics3DTools.ts

```diff
@@ -0,0 +1,51 @@
+namespace gdjs {
+  export namespace physics3d {
+    export const objectsCollide = function (
```

Comment:
nit: always use a verb to start the name of a function:

```suggestion
    export const areObjectsColliding = function (
```

Reply:
Should I change it for the 2D one too?

Reply:
Let's do it. This is not part of a documented API surface.